pkg/goversion/compat.go

```diff
@@ -14,7 +14,7 @@ var (
 	MinSupportedVersionOfGoMajor = 1
 	MinSupportedVersionOfGoMinor = 21
```

Comment:
While we're here we should also update this one.

Reply:
Done